frontend/src/components/pages/chat/components/chatList/ChatList.tsx

```diff
@@ -28,26 +18,38 @@ export const ChatList: React.FC = () => {
   };
 
   useEffect(() => {
+    const getChats = async () => {
+      setIsLoading(true);
+      const user = await getUserFromStorage();
+      const dialogs = await getAllDialogs(user._id);
+      const newDialogs = dialogs.map(
+        (dialog: any): DialogInfoProps => {
+          if (user._id !== dialog.participants[1]._id) {
+            return {
+              dialogId: dialog._id,
+              photo: dialog.participants[1].avatar,
+              name: `${dialog.participants[1].firstName} ${dialog.participants[1].lastName}`,
+              text: dialog.messages[0].body,
+              date: dayjs(dialog.messages[0].date).format('DD.MM.YY')
+            };
+          }
+          return {
+            dialogId: dialog._id,
+            photo: dialog.participants[0].avatar,
+            name: `${dialog.participants[0].firstName} ${dialog.participants[0].lastName}`,
+            text: dialog.messages[0].body,
+            date: dayjs(dialog.messages[0].date).format('DD.MM.YY')
+          };
+        }
+      );
+      const results = newDialogs.filter((chat: DialogInfoProps) =>
+        chat.name.toLowerCase().includes(search.toLowerCase())
+      );
+      setSearchResults(results);
+      setIsLoading(false);
```

Comment:
since we newer use search on BE no any reason to fetch new dialogs on search. Should be separate useEffect for filtering only